resources/assets/js/helpers.js

```diff
@@ -0,0 +1,4 @@
+export function copyToClipboard(eId) {
+  let e = document.getElementById(eId);
+  navigator.clipboard.writeText(e.innerHTML);
+}
```

Comment:
Make sure to add a story to come back to this.

Reply:
Added a story, #225